components/movementsensor/gpsnmea/component.go

```diff
@@ -96,6 +96,11 @@ func (g *NMEAMovementSensor) ReadSatsInView(ctx context.Context) (int, error) {
 	return g.cachedData.ReadSatsInView(ctx)
 }
 
+// ReadSatsInUse returns the number of satellites in use.
+func (g *NMEAMovementSensor) ReadSatsInUse(ctx context.Context) (int, error) {
+	return g.cachedData.ReadSatsInUse(ctx)
+}
+
```

Comment:
Not for this pr, but could we add following helper in gpsutils:
`map, err = g.CachedData.GetCommonReadings` -> outputs Fix, Sats, all the exported functions that we stuff into just readings for the nmea movement sensor.

That way we don't have a weird  dependency of a private functions in gpsrtk -> public functions in gpsnmea/component.go -> public functions in gpsutils/cachedData.go.

In my opinion, the `ReadFix`, `ReadSatsInView`, and `ReadSatsInUse` helpers should disappear, both from the nmeaSensor interface and the Readings function in both drivers. Just get the data from the cache, we already parse and store it there.